### DISCUSSION TITLE
NO Aircraft trail visibility when zoomed in

### DISCUSSION THREAD

**Discussion by BattleOverlord** [albar965/littlenavmap#1161] [Q&A]
I've recently made update to 3.0.6 verison 64 bit - MSFS. After this update I have troubles to see my aircraf trail when zoomed in. I have to be very very far (like satelite view of a continent) to be able see the aircraft trail. Before this update I was able to see aircraft trail no matter how much I was zoomed out/in. Went through topics here, through manual. There is no guide how to fix it. I would like to know how to do that, because I tried different colurs, different width, different type of aircraft trail nothing shows them to me when I'm zoomed in.
3 different aircraft trails. From Canada to Caribbean- From Ethiopia to Magagascar and from New Zealand to Tonga. The aircraft trail from Canada I can see only when zoomed out to crazy values like 1000-2000 nm. The aircraft trail from Ethiopia I can see up to 118Nm zoom, another zoom level it is gone. The third aircraft trail from New Zealand lasts up to closest zoomed in values (works for me as intended and like it did for me before update).
![118nm](https://github.com/albar965/littlenavmap/assets/173011014/e627b730-d644-478d-ad59-f37605c7539b)
![7,4nm](https://github.com/albar965/littlenavmap/assets/173011014/0bbfe49e-19be-4409-9bc9-9d899772825e)
![1884nm](https://github.com/albar965/littlenavmap/assets/173011014/a1c82bf4-b87d-4d56-b56d-02842cec484c)

So where can I find setting to make my aircraft trail visible when zoomed it to low values?

Thanks

**Comment by albar965** [comment]
I think this is already fixed with #1126 for the next release.
Can you attach the trail as GPX exported to this discussion? I'd like to verify if your case is fixed too.

**Reply by BattleOverlord** [reply]
Hope it is this. Just to clarify I use setting to show 100 000 - 150 000 aircraft trail point. It basically shows me my flights after one week. It is usually over 24 hours total. If some flight trails are not complete thats because when I updated LNAVMAP it went from 120k limit to the default number (dont remember probably 30k?) I cant share the file here directly - unsupported file extension. I used windows in-built .zip
[IFR Ivato (FMMI) to Ankazobe (TN).zip](https://github.com/user-attachments/files/15872341/IFR.Ivato.FMMI.to.Ankazobe.TN.zip)

Thanks for investigating :)

**Reply by albar965** [reply]
Thank you for the trail. I see no issues here with my 3.0.7.rc1 (not released yet). Zooming far and close and still see trails.
In any case let me know if you still see issues once it's released.

Alex

**Reply by BattleOverlord** [reply]
> Thank you for the trail. I see no issues here with my 3.0.7.rc1 (not released yet). Zooming far and close and still see trails. In any case let me know if you still see issues once it's released.
> 
> Alex

I just finished flight into Canada from Caribbean and it is visible at every level. My earlier flight today from Madagascar to Ethiopia is visible only when zoomed out to 1884nm. When I zoom closer it disappears. I tried the black dashed line from now which was there in the past. Maybe I did something wrong during update? I dont know. I downloaded the .rar file and manually deleted all the files and extracted the .rar into empty folder. I think I did it correclty. Anyway never had issues with aircraft trail in older versions of LNAVMAP. Looking forward to try new version in the future :)

**Reply by albar965** [reply]
The problem appears only for very long trails or flight plans like around-the-world plans and it happens only at certain zoom levels. 

BTW:The map gets really jerky for huge trails and I usually delete them before each flight.
The trails for all flights are also stored in the logbook where you can look at them and export them to GPX.

**Reply by BattleOverlord** [reply]
> The problem appears only for very long trails or flight plans like around-the-world plans and it happens only at certain zoom levels.
> 
> BTW:The map gets really jerky for huge trails and I usually delete them before each flight. The trails for all flights are also stored in the logbook where you can look at them and export them to GPX.

Thank you for mentioning the logbook. Completely forgot about this feature. Really awesome tool. I was able to load my old flight and export it and load it to the map :) using "append" function. Funnily enough after I added this old aircraft trail path to my current lnavmap all the routes I had problem with are now visible except the one from/to the Caribbean. I captured short video about this. My laymans opinion is the closer I get to heavy icons area the path starts to disappear at more zoomed out levels. It is visible in this video.

https://www.youtube.com/watch?v=VJJcvZIBD_E

**Reply by albar965** [reply]
Thanks for the video. I tried with the zoom distance and your trail shown in the video and it really seems it is fixed now.
Alex

**Reply by BattleOverlord** [reply]
![tv bug](https://github.com/user-attachments/assets/f1ff6df0-0ab8-45de-a711-f6d4b8d9b1a9)

Hello. I've discovered some weird interface bug. Only thing I changed was that I've connected also 4K TV to my GPU as "2nd monitor". Weird thing is that the TV is off and I dont even use it as 2nd monitor right now. I have a TV broadcast there. All the opened tabs on the sides are large, zoomed it and also the text is huge. It is like the whole LNAVMAP app is zoomed in. 
btw I did not want to report it in new topic so I put it here for now.

**Reply by albar965** [reply]
Try with options page `Display and Text` `High DPI ...` on or off.
Or set a smaller font for whole LNM on the same page.
Alex
